Sources/Swift/Integrations/UserFeedback/SentryFeedback.swift

```diff
@@ -3,32 +3,41 @@ import Foundation
 
 @objcMembers
 class SentryFeedback: NSObject {
-    enum Source: String {
+    @objc enum SentryFeedbackSource: Int, CustomStringConvertible {
+        public var description: String {
+            switch self {
+            case .widget: return "widget"
+            case .custom: return "custom"
+            }
+        }
+        
         case widget
         case custom
     }
     
     var name: String?
     var email: String?
     var message: String
-    var source: Source
+    var source: SentryFeedbackSource
     let eventId: SentryId
     
-    /// PNG data for the screenshot image
-    var screenshot: Data?
+    /// Data objects for any attachments. Currently the web UI only supports showing one attached image, like for a screenshot.
+    private var attachments: [Data]?
     
     /// The event id that this feedback is associated with, like a crash report.
-    var associatedEventId: String?
+    var associatedEventId: SentryId?
```

Comment:
Seemed to make more sense to use a more strict type here, not sure why i didn't do it before, maybe something didn't compile before, but it works here now